src/components/AppRepoForm/index.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
```

Comment:
Should the AppRepoForm be a modal like the Chart Deployment and Service Provision views? Feels a bit inconsistent.